src/components/Proposal/ProposalSummary/ProposalSummaryCO2ken.tsx

```diff
@@ -16,6 +17,14 @@ interface IProps {
 
 export default class ProposalSummaryCO2ken extends React.Component<IProps, null> {
 
+  private rawCallData(proposalState: IGenericPluginProposalState) {
+    return <>
+      <p>Raw call data:</p>
```

Comment:
Please use `i18next` here and in the rest of the places.

Reply:
Done